src/components/ConsoleIcons/index.tsx

```diff
@@ -198,10 +218,11 @@ export default function ConsoleLabel({name}): ReactElement {
     case 'applicationusers':
       return (
         <>
-          <ConsoleIconWrapper icon={ConsoleIcons.appUsers} /> <b>Users</b>
+          <ConsoleIconWrapper icon={ConsoleIcons.appUsers} />{' '}
+          <b>Application users</b>
         </>
       );
-    case 'viewprofile':
+    case 'viewappuserprofile':
```

Comment:
Have you checked if it's used already? If so, the files using this case must be updated :)

Reply:
Not from what I can see, unless I'm missing something?